### ISSUE TITLE
Speed limit | Prohibited/permitted site

### ISSUE THREAD

**Issue by Buckley27** [donaldzou/WGDashboard#427]
My idea I would like to add. 

Through Peer “Edit”
 And reduce the speed for example to 1 mbps or 5 mbps and so on.

Also the site or can IP addresses
For example, http://www.youtube.com/ limit to 5 mbps.
Any site specified called and limit. I think it will be very convenient

**Comment by donaldzou** [comment]
Yes, there's another use also pitched to me this idea. I'm looking to add this into WGDashboard ;)
